ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -66,7 +44,7 @@ public void WhenInitialized_FindCellStateOfCell(CellState state, int x, int y)
         [InlineData("XXX", "OXX", "XXX", 1, 1, 1 )]
         [InlineData("OOO", "OOO", "OOO", 1, 1, 8)]
         [InlineData("OXX", "XXX", "XXO", 1, 1, 2)]

```

Comment:
Given the deletion of tests for neighbour locations, we should add more tests cases here to cover the correct counting boundary cells. Potentially split into `ReturnCountOfLiveNeighbours_GivenBoundaryCoordinates` and `ReturnCountOfLiveNeighbours_GivenNonBoundaryCoordinates`